### ISSUE TITLE
StopPoints has no attribute 'StopPoint'

### ISSUE THREAD

**Issue by HTenkanen** [HTenkanen/transx2gtfs#14]
Should handle following:

```
"""
Traceback (most recent call last):
  File "C:\Hyapp\Anaconda3\lib\multiprocessing\pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "C:\Hyapp\Anaconda3\lib\multiprocessing\pool.py", line 44, in mapstar
    return list(map(*args))
  File "C:\HY-DATA\hentenka\KOODIT\Uni\transxchange_gtfspy\transx2gtfs\converter.py", line 150, in process_files
    stop_data = get_stops(data, naptan_stops_fp=naptan_stops_fp)
  File "C:\HY-DATA\hentenka\KOODIT\Uni\transxchange_gtfspy\transx2gtfs\stops.py", line 54, in get_stops
    for p in data.TransXChange.StopPoints.StopPoint:
  File "C:\Hyapp\Anaconda3\lib\site-packages\untangle.py", line 83, in __getattr__
    "'%s' has no attribute '%s'" % (self._name, key)
AttributeError: 'StopPoints' has no attribute 'StopPoint'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\HY-DATA\hentenka\KOODIT\Uni\transxchange_gtfspy\transx2gtfs\converter.py", line 254, in convert
    pool.map(process_files, workers)
  File "C:\Hyapp\Anaconda3\lib\multiprocessing\pool.py", line 266, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "C:\Hyapp\Anaconda3\lib\multiprocessing\pool.py", line 644, in get
    raise self._value
AttributeError: 'StopPoints' has no attribute 'StopPoint'
>>>
```

**Comment by aclong** [comment]
I got this error as well

**Comment by HTenkanen** [comment]
@aclong Yep, fix is on the way soon! Found also one other issue that is fixed (the last stop of a trip was always missing from the data).

**Comment by HTenkanen** [comment]
Fixed in #16
